docs/source/04_application_hooks.rst

```diff
@@ -0,0 +1,28 @@
+.. _user_interface_hooks-section:
+
+Application Hooks
+====================
+
+The application hooks allow plugins to add custom models or custom checks in the application as well to configure internal
```

Comment:
remove "... as well to configure internal settings from the ALFAsim.", I think it is referring to the Solver Configuration Hooks, in which were changed to the next new section.